### ISSUE TITLE
[invalid] returns_df.RData seems not right

### ISSUE THREAD

**Issue by TianYuJiang89** [Robot-Wealth/intraday-fx-seasonality-app#2]
in returns_df,  there're 21 tickers.

7 "USD" base tickers: "AUDUSD", "NZDUSD", "CHFUSD", "EURUSD", "GBPUSD", "JPYUSD", "CADUSD"
7 "EUR" base tickers: "AUDEUR", "NZDEUR", "CHFEUR", "GBPEUR", "JPYEUR", "USDEUR", "CADEUR"
7 "JPY" base tickers: "AUDJPY", "NZDJPY", "CHFJPY", "EURJPY", "GBPJPY", "USDJPY", "CADJPY"

First, we look at the USD base tickers.
"EURUSD" for example:
```
returns_df %>% filter(Ticker=="EURUSD") %>% head(1)
```
output:
```
# Groups:   Ticker [1]
  Ticker datetime             hour  year month   day  returns
  <chr>  <dttm>              <int> <dbl> <dbl> <int>    <dbl>
1 EURUSD 2009-01-02 07:00:00     1  2009     1     2 -0.00173
```
2009-01-02 07:00:00 UTC, convert to EST is 2009-01-02 02:00:00
But your hour is 1.
So we assume:
1. our "datetime" field is right alignment, it means the return from 2009-01-02 06:00:01 UTC to 2009-01-02 07:00:00 UTC is -0.00173
2. while our "hour" field is left alignment, it means the return from 2009-01-02 01:00:00 EST to 2009-01-02 01:59:59 EST is -0.00173

so it makes sense, and the rest of all USD base tickers have the same problem as EURUSD.

Then, we look at the EUR base tickers.
```
returns_df %>% filter(Ticker=="USDEUR") %>% head(1)
```
output:
```
# Groups:   Ticker [1]
  Ticker datetime             hour  year month   day returns
  <chr>  <dttm>              <int> <dbl> <dbl> <int>   <dbl>
1 USDEUR 2009-01-02 07:00:00     7  2009     1     2 0.00173
```
There's no conversion needed.
In USD base tickers, "datetime" is left alignment while the "hour" is right alignment.
But in EUR base tickers, datetime's hour is the same as the "hour" field.
The logic is not consistent.
The rest of all EUR base tickers have the same problem.

Last, we look at the JPY base tickers.
```
returns_df %>% filter(Ticker=="USDJPY") %>% head(1)
```
output:
```
# Groups:   Ticker [1]
  Ticker datetime             hour  year month   day  returns
  <chr>  <dttm>              <int> <dbl> <dbl> <int>    <dbl>
1 USDJPY 2009-01-02 07:00:00    15  2009     1     2 0.000241
```
2009-01-02 07:00:00 UTC, convert to EST is 2009-01-02 16:00:00
So, it's the same rule as USD base tickers.
The rest of all JPY base tickers is the same.

In conclusion:
All USD and JPY base tickers use the same logic:
The "datetime" is right alignment while the "hour" is left alignment.
But in EUR base tickers, both fields are right alignment.

**Comment by robotmasterkris** [comment]
Thanks for contributing!

I agree with everything you’ve said, except that the EUR tickers are inconsistent with the others. 

In relation to the EUR tickers, you said:

> There's no conversion needed

But a conversion is needed - CET is one hour ahead of UTC normally and two hours when daylight savings time is in effect. 

When you account for that conversion, all the pairs have consistent labelling.

**Comment by TianYuJiang89** [comment]
Sorry, my mistake, I thought CET and UCT is the same thing.
Thanks for the explanation, I'm clear now.
